crowdsource/views.py

```diff
@@ -0,0 +1,81 @@
+from django.shortcuts import render
+
+from django.db.models import Q
+from opencivicdata.legislative.models import Bill, VoteEvent
+
+from .issues import IssueType
+from .models import CrowdSourceIssue, CrowdSourceIssueResolver
+from .forms import IssueForm, ResolverForm
+from django.views.decorators.csrf import csrf_exempt
+from django.contrib import messages
+
+
+def reporter_utility(issue_type, identifier):
```

Comment:
this function name isn't going to be useful, something like get_object_by_identifier might be more helpful

Reply:
agreed